server/controllers/article.js

```diff
@@ -0,0 +1,39 @@
+import slug from 'slug';
+import articleRepo from '../repository/articleRepository';
+import { goodHttpResponse, badHttpResponse } from '../utilities/httpResponse';
+/**
+ * Article Controller class
+ */
+class Article {
+  /**
+   * Create a new article
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} User Object
+   */
+  static async createArticle(request, response) {
+    const {
+      title,
+      body,
+      description,
+      images,
+    } = request.body;
+    const readtime = Math.floor(body.split(' ').length / 3);
```

Comment:
why 3?

Reply:
From my research, the average person reads 180 words per minute on a monitor https://en.wikipedia.org/wiki/Words_per_minute. So that's 3 words per second on the average.